### ISSUE TITLE
Crash on iOS 8

### ISSUE THREAD

**Issue by yosimasu** [getsentry/react-native-sentry#114]
Environment:
- iOS: 8.1
- react-native: 0.42.3
- react-native-sentry: 0.12.12

Steps:
1. Create a react-native project, run it on a 8.1 simulator. -> work
2. Add react-native-sentry, run react-native link, configure, and run it again. -> app crash

I just got the following logs:
```
Jun 21 11:47:24 MBP com.apple.CoreSimulator.SimDevice.12C7D028-494E-420B-87D6-23C4F20C2F04.launchd_sim[2985] (UIKitApplication:org.reactjs.native.example.SentryTest[0x32c2][3139]) <Notice>: Service exited due to signal: Trace/BPT trap: 5
Jun 21 11:47:24 MBP.local SpringBoard[2996] <Warning>: Application 'UIKitApplication:org.reactjs.native.example.SentryTest[0x32c2]' crashed.
Jun 21 11:47:24 MBP assertiond[3000] <Error>: assertion failed: 16F73 12B411: assertiond + 11523 [3F572A0B-7E12-378D-AFEE-EA491BAF2C36]: 0x1
```

**Comment by HazAT** [comment]
This should be fixed with the latest release `> 0.13.0`

**Comment by yosimasu** [comment]
Hi @HazAT 

I have tested v0.14.0 on the same project.
It has crashed.
Following is the startup log from my simulator:

```
Jun 30 12:17:34 MASU-MBP assertiond[16760] <Error>: assertion failed: 16F73 12B411: assertiond + 11523 [3F572A0B-7E12-378D-AFEE-EA491BAF2C36]: 0x1
Jun 30 12:17:34 MASU-MBP Unknown[16760] <Error>:
Jun 30 12:17:34 MASU-MBP.local SpringBoard[16756] <Warning>: LICreateIconForImage passed NULL CGImageRef image
Jun 30 12:17:34 MASU-MBP com.apple.CoreSimulator.SimDevice.12C7D028-494E-420B-87D6-23C4F20C2F04.launchd_sim[16745] (com.apple.imfoundation.IMRemoteURLConnectionAgent) <Error>: The _DirtyJetsamMemoryLimit key is not available on this platform.
Jun 30 12:17:34 MASU-MBP SentryTest[27377] <Error>: assertion failed: 16F73 12B411: libxpc.dylib + 59417 [464E62EA-4CF2-3FEF-9C17-692AD8D66AA8]: 0x7d
Jun 30 12:17:34 MASU-MBP Unknown[27377] <Error>:
Jun 30 12:17:34 MASU-MBP assertiond[16760] <Error>: assertion failed: 16F73 12B411: assertiond + 11523 [3F572A0B-7E12-378D-AFEE-EA491BAF2C36]: 0x1
Jun 30 12:17:34 MASU-MBP Unknown[16760] <Error>:
Jun 30 12:17:34 MASU-MBP assertiond[16760] <Error>: assertion failed: 16F73 12B411: assertiond + 11523 [3F572A0B-7E12-378D-AFEE-EA491BAF2C36]: 0x1
Jun 30 12:17:34 MASU-MBP Unknown[16760] <Error>:
Jun 30 12:17:34 MASU-MBP.local SpringBoard[16756] <Warning>: [MPUSystemMediaControls] Updating supported commands for now playing application.
Jun 30 12:17:35 MASU-MBP.local SentryTest[27377] <Notice>: Initializing <RCTBatchedBridge: 0x7ffe22e579f0> (parent: <RCTBridge: 0x7ffe22e779a0>, executor: RCTJSCExecutor)
Jun 30 12:17:35 MASU-MBP.local SentryTest[27377] <Warning>: Sentry Started -- Version: 3.1.2
Jun 30 12:17:35 MASU-MBP.local SentryTest[27377] <Notice>: Running application SentryTest ({
	    initialProps =     {
	    };
	    rootTag = 1;
	})
Jun 30 12:17:35 MASU-MBP.local SentryTest[27377] <Notice>: Running application "SentryTest" with appParams: {"rootTag":1,"initialProps":{}}. __DEV__ === true, development-level warning are ON, performance optimizations are OFF
Jun 30 12:17:35 MASU-MBP com.apple.CoreSimulator.SimDevice.12C7D028-494E-420B-87D6-23C4F20C2F04.launchd_sim[16745] (UIKitApplication:org.reactjs.native.example.SentryTest[0xda01][27377]) <Notice>: Service exited due to signal: Trace/BPT trap: 5
Jun 30 12:17:35 MASU-MBP com.apple.CoreSimulator.SimDevice.12C7D028-494E-420B-87D6-23C4F20C2F04.launchd_sim[16745] (UIKitApplication:org.reactjs.native.example.SentryTest[0xda01]) <Notice>: Service only ran for 0 seconds. Pushing respawn out by 2147483647 seconds.
Jun 30 12:17:35 MASU-MBP.local SpringBoard[16756] <Warning>: Application 'UIKitApplication:org.reactjs.native.example.SentryTest[0xda01]' crashed.
Jun 30 12:17:35 MASU-MBP.local assertiond[16760] <Warning>: notify_suspend_pid() failed with error 7
Jun 30 12:17:35 MASU-MBP assertiond[16760] <Error>: assertion failed: 16F73 12B411: assertiond + 11523 [3F572A0B-7E12-378D-AFEE-EA491BAF2C36]: 0x1
Jun 30 12:17:35 MASU-MBP Unknown[16760] <Error>:
Jun 30 12:17:35 MASU-MBP assertiond[16760] <Error>: assertion failed: 16F73 12B411: assertiond + 11523 [3F572A0B-7E12-378D-AFEE-EA491BAF2C36]: 0x1
Jun 30 12:17:35 MASU-MBP Unknown[16760] <Error>:
```
